test/passes/fuzz-exec_O.txt

```diff
@@ -1,7 +1,5 @@
 [fuzz-exec] calling $func_0
-[trap final > memory: 18446744073709551615 > 65514]
 [fuzz-exec] calling $func_1
```

Comment:
what happened here?

Reply:
I printed `[trap]` into not stdout but stderr, because it's more like errorneous event..? Do you want it back to stdout? If so, I think we can add that to the expected output as well.

Reply:
Oh, I missed that in the PR description. Sounds fine.